spiffworkflow-frontend/dev.Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:20.8.1-bookworm-slim AS base
+
+WORKDIR /app
+
+COPY . /app/
+
+RUN npm i
```

Comment:
This is just for local development so node, etc doesn't have to be installed on the host. I was thinking that when the package files changed you would just rebuild the image since that is all this Dockerfile is doing.

Reply:
Reconsidered this and now am just copying the package json files before running `npm i`.